settings.php

```diff
@@ -742,6 +748,47 @@
         // Add tab to settings page.
         $page->add($tab);
 
```

Comment:
At this point we have to add the $page (referring to 'theme_boost_union_functionality') to the category, before creating a new page (theme_boost_union_blockregion):


```
// Add settings page to the admin settings category.
$ADMIN->add('theme_boost_union', $page);
```

Without this there is a section error thrown, when trying to open the functionality seetings